src/main/resources/twilio-node/responseModel.mustache

```diff
@@ -1,11 +1,9 @@
 {{#responseModel}}
-{{#hasEnums}}
```

Comment:
Assuming this is not being used?

Reply:
It's just duplicative since we look for `isEnum` anyway.